src/main/java/org/craftercms/studio/model/rest/WriteConfigurationRequest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.model.rest;
+
+public class WriteConfigurationRequest {
+
+    private String siteId;
+    private String module;
+    private String location;
+    private String environment;
+    private String content;
```

Comment:
This is the whole XML right? Could there be any error when Jackson parses it from JSON?

Reply:
So far no errors found.

Reply:
Maybe it would be good to also test yaml content as you mentioned it is planned for the future

Reply:
yaml not relevant at the moment

Reply:
I don't understand this part.

Reply:
I discussed with Dejan. We should be okay at this time. XML parsing errors will result in `bad request` back to caller.